bcl/inc/bc_radio.h

```diff
@@ -55,4 +57,6 @@ bool bc_radio_pub_co2(float *concentration);
 
 bool bc_radio_pub_buffer(void *buffer, size_t length);
 
+void bc_radio_button_pair(bc_button_t *button, bc_led_t *led);
```

Comment:
I would like to discuss the function name since it may not be very obvious what it does - how about `bc_radio_init_pairing_button` and `bc_radio_init_pairing_led` ?

Reply:
I definetely would like to rename the function. But I dont agree that the function needs to be separated for button and led. This is helper function for begineers with Core Module where is a button and LED. Advanced programmers should write their own button callback etc.
What I like is to create a one-liner. So lets try to find better name for button and led combo, but `bc_radio_init_pairing_button_and_led` is too long :)